src/components/CompareResults/SearchResultsItem.tsx

```diff
@@ -0,0 +1,45 @@
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableRow from '@mui/material/TableRow';
+
+import type { Revision } from '../../types/state';
+import {
+  truncateHash,
+  getLatestCommitMessage,
+  formatDate,
+} from '../../utils/helpers';
+
+export default function SearchResultsItem(props: SearchResultsItemProps) {
```

Comment:
why are we showing these results differently than in the first page? Can't we reuse the same styling?